src/main/java/lesson8WeekDaysNames/DayOfTheWeek.java

```diff
@@ -0,0 +1,8 @@
+package lesson8WeekDaysNames;
+
+public class DayOfTheWeek {
+
+    public static String whatDayIsIt(WeekDay day) {
```

Comment:
Niepoprawna nazwa metody.

Metoda zwraca nazwę dnia tygodnia, więc zapiszmy to jako "getDayName" lub jakoś podobnie.